proto/process/agent.proto

```diff
@@ -681,6 +767,7 @@ enum K8sResource {
     VERTICALPODAUTOSCALER = 22;
     HORIZONTALPODAUTOSCALER = 23;
     NETWORKPOLICY = 24;
+    ECSTASK = 150;
```

Comment:
This change matchs https://github.com/DataDog/datadog-agent/blob/076db95494f6cd8ee8d53fb469f44fb40b923c46/pkg/orchestrator/model/types.go#L83